test/uk/gov/hmrc/transitmovementsvalidator/services/ValidationServiceSpec.scala

```diff
@@ -76,7 +81,45 @@ class ValidationServiceSpec extends AnyFreeSpec with Matchers with MockitoSugar
       whenReady(result) {
         r =>
           r.isLeft mustBe true
-          r.left.get.head.isInstanceOf[SchemaValidationError]
+          r.left.get.head.isInstanceOf[XmlSchemaValidationError]
+      }
+    }
+  }
+
+  "On Validate JSON" - {
+    "when valid JSON is provided for the given message type, return a Right" in {
+      val source = FileIO.fromPath(Paths.get(s"$testDataPath/cc015c-generated-from-json-schema.json"))
+      val sut    = new ValidationServiceImpl
+      val result = sut.validateJSON(validCode, source)
+
+      whenReady(result) {
+        r =>
```

Comment:
possibly just _.isRight

Reply:
The test would always evaluate to true without the assertion